platform/pulp/tasking/tasks.py

```diff
@@ -96,7 +96,7 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
     try:
         celery.tasks[name].apply_async(*inner_args, **inner_kwargs)
```

Comment:
@BrnoPCmaniak I'm thinking of something like: https://github.com/werwty/pulp/commit/5c8178d7ac2637d9fd0b7eb6f53f6802a36525cd

Where additional arguments are being passed through options, and you would end up with `task(args=args, kwargs=kwargs, **options)`